pkgs/by-name/ho/horcrux/package.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "horcrux";
+  version = "0-unstable-2023-09-19";
```

I see that horcrux has a [v0.2 release](https://github.com/jesseduffield/horcrux/releases/tag/v0.2). @AndersonTorres should the version then be?

```suggestion
  version = "0.2-unstable-2023-09-19";
```

---

There's seemingly also [0.3](https://github.com/jesseduffield/horcrux/releases/tag/v0.3), so:

```suggestion
  version = "0.3-unstable-2023-09-19";
```

---

Interesting, @OPNA2608… Thanks for chiming in on this! I only checked the [releases](https://github.com/jesseduffield/horcrux/releases).

---

Good point @afh. @OPNA2608 thank you for the clarification!